Removed unused routes/controllers/views

werebus opened the pull request:
Forms
---------

Everything about modifying a `Form` other than destroying them is now done via drafts. The `#new` route doesn't load due to a missing template, the `#edit` route doesn't load either. I couldn't find any reference in the app to the `#create` or `#update` action at all. I removed all 4 of those routes and one spec.

Fields
--------
There used to be a mechanism for modifying the allowed "options" for a field through these two routes. It was removed in #99, leaving no need for this controller at all.

FormDrafts
---------------
There were two routes, `#move_field` and `#remove_field` in the routes file whose controller actions were also removed in #99.

Anbranin reviewed (approved):
this is great. I'm surprised coverage didn't change though! That is, if you ran rspec on your own machine. I see that there are no failures in Travis.

werebus commented:
Well, 100% is 100% I removed a useless controller, but that controller _was_ 100% covered. I removed some view files, but those aren't included in coverage calculations. And the rest was just changes to the routes file.
